
<launch>  
  <!--Astar server-->
    <!-- <include file="$(find astar)/launch/astar.launch">
          <arg name="map_size_x" value="100.0"/>
          <arg name="map_size_y" value="100.0"/>
          <arg name="map_size_z" value=" 0.2"/>
          <arg name="margin" value=" 0.0"/>
          <arg name="resolution" value=" 0.2"/>
          <arg name="inflate_num" value=" 1"/>
          <arg name="inflate_num_con_obs" value=" 5"/>
          <arg name="start_x" value=" 0.0"/>
          <arg name="start_y" value=" 0.0"/>
          <arg name="start_z" value=" 0.1"/>
          <arg name="odom" value="/t265/odom/sample"/>
          <arg name="obstacles_map" default="/obstacles_detection1/proj_obstacles"/>
          <arg name="pose" value="/global_pose"/>
          <arg name="goal" value="/planner_goal"/>
          <arg name="map_frameID" value="t265_odom_frame"/>
          <arg name="use_localmap" value="true"/>
     </include> -->

  <!-- Obstacles_detection -->
	<!-- <group ns="/rgbd_camera" >		 -->
		<node pkg="obstacles_detection" type="obstacles_detection1" name="obstacles_detection1" output="screen">
    	<!-- <remap from="cloud" to="/camera/depth/color/points"/> -->
			<!-- <remap from="obstacles" to="/planner_cloud"/> -->

			<param name="frame_id" type="string" value="camera_link"/>		
			<param name="map_frame_id" type="string" value="t265_odom_frame"/>
			<param name="wait_for_transform" type="bool" value="true"/>
			<param name="mapFrameProjection" type="bool" value="true"/>

      <param name="preVoxelFiltering" type="bool" value="true"/>		
			<param name="cellSize" type="double" value="0.05"/>		
			<param name="footprintLength" type="double" value="0.0"/>		
			<param name="footprintWidth" type="double" value="0.0"/>		
			<param name="footprintHeight" type="double" value="0.0"/>		
			<param name="minGroundHeight" type="double" value="0.0"/>		
			<param name="maxObstacleHeight" type="double" value="0.0"/>		
			<param name="normalsSegmentation" type="bool" value="true"/>		
			<param name="groundIsObstacle" type="bool" value="false"/>		
			<param name="maxGroundAngle" type="double" value="0.75"/>		
			<param name="clusterRadius" type="double" value="0.5"/>		
			<param name="flatObstaclesDetected" type="bool" value="true"/>		
			<param name="minClusterSize" type="int" value="10"/>		
			<param name="noiseFilteringRadius" type="double" value="0.0"/>		
			<param name="noiseFilteringMinNeighbors" type="int" value="5"/>		
			<param name="normalKSearch" type="int" value="8"/>		
			<param name="maxGroundHeight" type="double" value="0.0"/>		
  		</node>
	<!-- </group> -->
  <arg name="rviz_cfg"                default="$(find obstacles_detection)/launch/azimut3_stereo_nav.rviz" />
  <node pkg="rviz" type="rviz" name="rviz" args="-d $(arg rviz_cfg)"/>
  <!-- <include file="$(find publish_goal)/launch/publish_goal.launch">
  </include> -->

</launch>
